docs/.vitepress/config.mts

```diff
@@ -9,12 +9,17 @@ const reasonGrammar = JSON.parse(
   readFileSync(join(__dirname, "./reasonml.tmLanguage.json"), "utf8")
 );
 
+// https://github.com/ocamllabs/vscode-ocaml-platform/blob/master/syntaxes/dune.json
+const duneGrammer = JSON.parse(
```

Comment:
Is it "grammer" or "grammar"?

Reply:
:face_exhaling: Fixed in https://github.com/melange-re/melange-re.github.io/pull/188/commits/84b4e2e98f7340d6249c6084e69769939b04ee07